modules/apps/commerce/commerce-payment-service/src/main/java/com/liferay/commerce/payment/internal/integration/FunctionCommercePaymentIntegration.java

```diff
@@ -229,6 +236,13 @@ private JSONObject _getPayloadJSONObject(
 		String className = _portal.getClassName(
 			commercePaymentEntry.getClassNameId());
 
+		String sessionResult = ParamUtil.getString(
+			httpServletRequest, "sessionResult");
+
+		if (!Validator.isBlank(sessionResult)) {
+			commercePaymentEntryJSONObject.put("sessionResult", sessionResult);
```

Comment:
make this as much generic as possible, by sending all httpServletRequest params and not a specific one